src/org/traccar/reports/ReportUtils.java

```diff
@@ -46,6 +47,14 @@ public static String getSpeedUnit(long userId) {
         return unit != null ? unit : "kn";
     }
 
+    public static TimeZone getTimezone(long userId) {
+        String timezone = Context.getPermissionsManager().getUser(userId).getTimezone();
+        if (timezone == null) {
```

Comment:
I was mostly talking about handling user/server configuration, like we do in the web app. If we never used `forceSettings`, we should probably start doing it. I recommend creating another issue to fix all the places.